cpp/vm/evmzero/interpreter.cc

```diff
@@ -1487,6 +1489,24 @@ struct CreateImpl {
     if (gas -= mem_cost; gas < 0) [[unlikely]]
       return {.dynamic_gas_costs = initial_gas - gas};
 
+    if (ctx.revision >= EVMC_SHANGHAI) {
+      const int64_t kMaxCodeSize = 24576;
```

Comment:
maybe add a comment explaining where does this number comes from ?

Reply:
I don't think it makes sense to write a huge comment why something has been introduced and I also don't want to write a comment for a constant definition that it is from the ethereum standard.

Reply:
I think a reference to the source of this value would be in order. How do I find the definition of this value? 
just the eip number would be enough.

Reply:
I added the EIP number as a comment.